packages/react-scripts/backpack-addons/ssr/customWebpackUtils.js

```diff
@@ -78,13 +78,22 @@ function createCustomCompiler(
 
     forkTsCheckerWebpackPlugin
       .getCompilerHooks(compiler)
-      .receive.tap('afterTypeScriptCheck', (diagnostics, lints) => {
-        const allMsgs = [...diagnostics, ...lints];
+      .waiting.tap('awaitingTypeScriptCheck', () => {
```

Comment:
Plugin API changed, pulled this change from the upstream react-dev-utils example